packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -456,6 +463,11 @@ export class UnthemedSpanBarRow extends React.PureComponent<SpanBarRowProps> {
                     className={styles.errorIcon}
                   />
                 )}
+                {hasReference && (
```

Comment:
We're currently adding a few other features that link from traces to other data so I'd like to remove this icon for the moment until we have a more general approach for displaying links in the SpanBar. I think linking from the details view will be good for the moment.